synapseAnnotations/data/common/minimal_Sage_standard.json

```diff
@@ -1045,6 +1055,41 @@
         "value": "CD8+ T-Cells",
         "description": "",
         "source": ""
+      },
+      {
+        "value": "CRISPR arachnoid",
```

Comment:
Is this arachnoid with a CRISPR mutation of a gene?  If so, should probably be arachnoid and use the ONC annotations to apply the crispr info, no?